var/spack/repos/builtin/packages/py-pyseer/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyPyseer(PythonPackage):
+    """Python implementation of Sequence Element Enrichment Analysis (SEER)"""
+
+    homepage = "https://github.com/mgalardini/pyseer"
+
+    url = "https://github.com/mgalardini/pyseer/archive/refs/tags/1.3.11.tar.gz"
+
+    version("1.3.11", sha256="06ea2987509f9c1952bbb90e4b59c6f5a4f2ca9e88e7dac5f5cb7f43aa693a1b")
+
+    # build options
+    variant("matplotlib", default=True, description="Enables automatic scree plots")
+    variant("dendropy", default=True, description="Enables calculating phylogeny distances")
+    variant("kmer", default=True, description="Enables k-mer mapping and annotation")
+
+    # build deps
+    depends_on("py-setuptools", type="build")
+
+    # additional deps
+    # https://pyseer.readthedocs.io/en/master/installation.html
+    # also from
+    # github.com/bioconda/bioconda-recipes/blob/master/recipes/pyseer/meta.yaml
+    # the deps are messy and this project is not updated
+    depends_on("py-numpy@1.15.2:", type=("build", "run"))
+    depends_on("py-scipy@1.1.0:", type=("build", "run"))
+    depends_on("py-pandas@0.23.4:", type=("build", "run"))
+    depends_on("py-scikit-learn@0.20.0:", type=("build", "run"))
+    depends_on("py-statsmodels@0.9.0:", type=("build", "run"))
+    depends_on("py-pysam@0.15.3:", type=("build", "run"))
+    depends_on("py-glmnet-python@1.0", type=("build", "run"))
```

Comment:
According to [setup.py#L78](https://github.com/mgalardini/pyseer/blob/1.3.11/setup.py#L78)

Reply:
haha, incredible. It looks like pip and conda both download from the johnlees repo, but then all the homepages and source code link to the bbalasub1 repo, which is how I got mixed up:

https://anaconda.org/conda-forge/glmnet_py
https://github.com/conda-forge/glmnet_py-feedstock/blob/a32dd372c6e1a8cc25cd9dfd224777cebb15ff4b/recipe/meta.yaml#L2

I was wondering where conda-forge got v1.0.2 from. I'll make this change and have to confirm that it still works as expected.

Reply:
Yes, I also found it quite confusing. What makes it even worse is that both the bbalasub1 and johnlees repos claim that the packages can be installed with `pip install glmnet_py` but when you check that on pypi this is again an additional repo (namely hanfang) ...